playwright.config.ts

```diff
@@ -30,6 +30,7 @@ export default defineConfig({
 
     /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
     trace: "retain-on-failure",
+    video: "retain-on-failure",
```

Comment:
Czy chcemy na mainie mieć zapisywane video? Myślę, że ok, chyba ze na ci/cd by to nie pykło, ale to się okaże.

Reply:
a to mi się przypadkiem wkradło, to jutro dogadamy